README.md

```diff
@@ -9,194 +9,192 @@
+Becoming an FDC3 Participant is as easy as attending a meeting and/or raising issues for changes you'd like see in the Standard, commenting on issues others have raised or even asking questions (which can often result in the clarification of the Standard's documentation to help others with the same questions in future).
 
-Upon enrollment as an [FDC3 voting participant](https://github.com/orgs/finos/teams/fdc3-participants), you will be invited to join the [FINOS GitHub organization](https://github.com/orgs/finos/people) and the [fdc3-participants](https://github.com/orgs/finos/teams/fdc3-participants) GitHub team.
+If you are going to use, implement or benefit from the FDC3 Standard, you should accept the terms of the [Community Specification Contributor License Agreement](https://github.com/CommunitySpecification/Community_Specification/blob/main/.0_CS_Contributor_License_Agreement.md).  Accepting this agreement will allow you to vote on future changes to the standard.  To do this:
 
-#### Become an FDC3 Editor or Maintainer
-Once you are an enrolled participant in FDC3, you can apply to become an editor or maintainer by contacting the existing FDC3 maintainers at [fdc3-maintainers@finos.org](mailto:fdc3-maintainers@finos.org) and then seeking the approval of the FDC3 Standards Working Group. Generally, the maintainers will look for both a history of contribution to FDC3 and a commitment to investing sufficient time in the role from any prospective candidates before proposing them to the Standards Working Group for approval. If you are new to FDC3, but willing to make the investment of time, the maintainers can work with you to build up a history of contribution.
+1. **Add your details to the [NOTICES.md](NOTICES.md) file in the "License Acceptance Section".**  [Click here to edit the file and create a pull request](https://github.com/finos/FDC3/edit/master/NOTICES.md).  This will begin the process of creating a pull request.  
+2. **Click the "Create Pull Request" button.** This will notify the FDC3 team of your application.  
+3. **Review your EasyCLA settings on the Pull Request**.  Within the Pull Request, you will have a button to review your CLA settings.  A completed CLA will look like the one below:
 
-#### Mailing list
-You can contact the FDC3 maintainers by sending an email to [fdc3-maintainers@finos.org](mailto:fdc3-maintainers@finos.org).
+<img width="932" alt="EasyCLA" src="https://user-images.githubusercontent.com/568673/195820700-a7978380-63cb-48c3-a1c2-63d4586d6bf9.png">
 
-# FDC3 Archive
-An archive of FDC3 documentation and meeting notes is available at https://finosfoundation.atlassian.net/wiki/spaces/FDC3/overview. The mailing list archive for fdc3@finos.org is available at https://groups.google.com/a/finos.org/g/fdc3
+- For more details on the Contributor License Agreement, please see the [FINOS CLA Page](https://community.finos.org/docs/governance/software-projects/easycla/). 
```

Comment:
@robmoffat @scooter99boston again, shouldn't this be more specific that it's not a FINOS CLA, rather a CSL CLA that will be signed?

In other words, if they have a FINOS (Apache) CCLA/ICLA signed, it will have no bearing on FDC3 and they need to follow the procedure above. 

I think this needs to be crystal clear. Maybe add a specific NOTE on it.

Reply:
yes, I think this needs to change.  At the moment, I still don't have the PDFs for the ICLA or CCLA under CSL.  I'll link these when I get them added on EasyCLA.

Reply:
Ok, I've tidied this up and linked to the EasyCLA docs too.  Added your note.